main/main.go

```diff
@@ -155,3 +164,28 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func restoreIBU(client client.Client) error {
+	ibu := &lcav1alpha1.ImageBasedUpgrade{}
+	filePath := filepath.Join(utils.Host, utils.IBUFilePath)
```

Comment:
Wondering why we are doing it here? 
Why not to do it in post-pivot phase? I don't think this is actually the right place. 
I think we should have everything in /opt/openshift folder and it should be cleaned up at the end of the process.

Reply:
I think we are creating duplication logic for mostly same operations. 
We should decide or we are pushing everything after LCA operator start or we are pushing it in in post-pivot part.
Another way we will never be able to understand the code.

Reply:
This is because it's a two step process. I don't want the controller to try reconciling the result from the create without the status. The post-pivot command is probably running in parallel when the controller pod is brought up by kubelet. I actually prefer not to do it here, only if there is a way to import the CR with its status in one atomic operation.

Reply:
@donpenney @jc-rh we need to decide. For sure we should not have code in 2 different places that does the same.
So we have 2 options here: 
1. Post-pivot is going to do all the apply/cleanup logic as it does now
2. Operator should do it while start running. 

Another way it will be not manageable at all. We can meet and decide about design one more time but we should understand where we implement each part of the logic as currently it is mess

Reply:
I believe i need to understand why there is an issue with applying status from post-pivot code

Reply:
After talking with @jc-rh i understood why we need it here